pci.go

```diff
@@ -18,46 +20,14 @@ var (
 	)
 )
 
-type PCIProgrammingInterface struct {
-	Id   string // hex-encoded PCI_ID of the programming interface
-	Name string // common string name for the programming interface
-}
-
-type PCISubclass struct {
-	Id                    string                     // hex-encoded PCI_ID for the device subclass
-	Name                  string                     // common string name for the subclass
-	ProgrammingInterfaces []*PCIProgrammingInterface // any programming interfaces this subclass might have
-}
-
-type PCIClass struct {
-	Id         string         // hex-encoded PCI_ID for the device class
-	Name       string         // common string name for the class
-	Subclasses []*PCISubclass // any subclasses belonging to this class
-}
-
-// NOTE(jaypipes): In the hardware world, the PCI "device_id" is the identifier
-// for the product/model
-type PCIProduct struct {
-	VendorId   string        // vendor ID for the product
-	Id         string        // hex-encoded PCI_ID for the product/model
-	Name       string        // common string name of the vendor
-	Subsystems []*PCIProduct // "subdevices" or "subsystems" for the product
-}
-
-type PCIVendor struct {
-	Id       string        // hex-encoded PCI_ID for the vendor
-	Name     string        // common string name of the vendor
-	Products []*PCIProduct // all top-level devices for the vendor
-}
-
 type PCIDevice struct {
 	Address              string // The PCI address of the device
-	Vendor               *PCIVendor
-	Product              *PCIProduct
-	Subsystem            *PCIProduct // optional subvendor/sub-device information
-	Class                *PCIClass
-	Subclass             *PCISubclass             // optional sub-class for the device
-	ProgrammingInterface *PCIProgrammingInterface // optional programming interface
+	Vendor               *pcidb.PCIVendor
+	Product              *pcidb.PCIProduct
+	Subsystem            *pcidb.PCIProduct // optional subvendor/sub-device information
```

Comment:
go fmt somehow not fmt the comments on the same spacing as below?

Reply:
yeah, weird, huh? I have go fmt run when I save in vim and for some reason it's lining up the types but not the comments. oh well...